backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/utils/MapCsvToStore.kt

```diff
@@ -0,0 +1,11 @@
+package app.ehrenamtskarte.backend.stores.utils
+
+import app.ehrenamtskarte.backend.stores.COUNTRY_CODE
+import app.ehrenamtskarte.backend.stores.importer.common.types.AcceptingStore
+import app.ehrenamtskarte.backend.stores.webservice.schema.types.CSVAcceptingStore
+
+fun mapCsvToStore(csvStore: CSVAcceptingStore): AcceptingStore {
```

Comment:
optional:
I think there is another nice place for this function - data class AcceptingStore
we could add a companion object like this:
```
    companion object {
        fun fromCSV(csvStore: CSVAcceptingStore): AcceptingStore {
            // TODO
        }
    }
```
and it will be like a static factory function for AcceptingStore class